diff --git a/node_modules/@storybook/addon-react-native-web/preset.js b/node_modules/@storybook/addon-react-native-web/preset.js
index 9180220..555ab52 100644
--- a/node_modules/@storybook/addon-react-native-web/preset.js
+++ b/node_modules/@storybook/addon-react-native-web/preset.js
@@ -65,7 +65,7 @@ module.exports = {
             '@babel/preset-react',
             'module:metro-react-native-babel-preset',
           ],
-          plugins: ['@babel/plugin-proposal-class-properties', ...babelPlugins],
+          plugins: [...babelPlugins],
         },
       });
     }
